bin/inspect_all_known_projects.py

```diff
@@ -62,18 +62,14 @@ def __init__(self, cached_file: Path | str, *, online: bool) -> None:
     def get(self, repo: str, filename: str) -> str | None:
         if self.online:
             try:
-                self.contents[filename][repo] = (
-                    GhApi(*repo.split("/")).get_content(filename).decode()
-                )
+                self.contents[filename][repo] = GhApi(*repo.split("/")).get_content(filename).decode()
```

Comment:
Yes. But we probably have black (with a smaller maximum line length) to blame for the version we now have, already?

Reply:
The other version is fine, too.

Reply:
It's a bit long, though. If it's a simple one-step concept (i.e., get the contents of this file in this repo), it's nice if it fits on one line.

Reply:
Not if that line can't be read in your editor without scrolling. Editors don't widen out when you come to a longer line just because it's a single concept.